lib/shopify_app.rb

```diff
@@ -58,4 +58,7 @@ def self.use_webpacker?
   require 'shopify_app/session/session_storage'
   require 'shopify_app/session/shop_session_storage'
   require 'shopify_app/session/user_session_storage'
+
+  # omniauth_configuration
```

Comment:
Unnecessary comment?

Reply:
We wanted to be categorize the `OmniAuthConfiguration` class similar to other modules and classes above.

Reply:
We followed the convention of the file, which seems to have similar comments breaking requirements apart. Do you think this belongs with an existing collection within the same file?

Reply:
Ah, I was only seeing the excerpt. Let's stick with that convention then.